### ISSUE TITLE
Hi hsientzucheng 

### ISSUE THREAD

**Issue by hsuchengmath** [hsientzucheng/MKR.PyTorch#1]
Hi My name is hsuchengmath . 
I recently research recommendation system and my Boss told me that 
I read MKR the paper . I had read and try to implement it by Pytorch . 
But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
After frustration , I search for MKR Pytorch on website and find your GitHub . 

Can we be friend ?

**Comment by neuliyiping** [comment]
> Hi My name is hsuchengmath .
> I recently research recommendation system and my Boss told me that
> I read MKR the paper . I had read and try to implement it by Pytorch .
> But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
> After frustration , I search for MKR Pytorch on website and find your GitHub .
> 
> Can we be friend ?

Hi My name is Yiping Li, recently  I  research recommendation system basing on KGE, do you know how can I get a list which contains such data:

Top 10 music data users like

 do you know ? hsuchengmath.

**Comment by hsientzucheng** [comment]
> Hi My name is hsuchengmath .
> I recently research recommendation system and my Boss told me that
> I read MKR the paper . I had read and try to implement it by Pytorch .
> But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
> After frustration , I search for MKR Pytorch on website and find your GitHub .
> 
> Can we be friend ?

Hi hsuchengmath, sorry for my late reply. Feel free to refer to my code.

**Comment by hsientzucheng** [comment]
> > Hi My name is hsuchengmath .
> > I recently research recommendation system and my Boss told me that
> > I read MKR the paper . I had read and try to implement it by Pytorch .
> > But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
> > After frustration , I search for MKR Pytorch on website and find your GitHub .
> > Can we be friend ?
> 
> Hi My name is Yiping Li, recently I research recommendation system basing on KGE, do you know how can I get a list which contains such data:
> 
> Top 10 music data users like
> 
> do you know ? hsuchengmath.

Hi @neuliyiping, do you mean to get an open dataset of music recommendation?
Here is one on Kaggle after quick googling: https://www.kaggle.com/c/kkbox-music-recommendation-challenge

**Comment by hsientzucheng** [comment]
Hi neuliyiping, do you mean to get an open dataset of music recommendation?
Here is one on Kaggle after quick googling: https://www.kaggle.com/c/kkbox-music-recommendation-challenge

**Comment by hsientzucheng** [comment]
> > Hi My name is hsuchengmath .
> > I recently research recommendation system and my Boss told me that
> > I read MKR the paper . I had read and try to implement it by Pytorch .
> > But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
> > After frustration , I search for MKR Pytorch on website and find your GitHub .
> > Can we be friend ?
> 
> Hi My name is Yiping Li, recently I research recommendation system basing on KGE, do you know how can I get a list which contains such data:
> 
> Top 10 music data users like
> 
> do you know ? hsuchengmath.

Hi neuliyiping, do you mean to get an open dataset of music recommendation?
Here is one on Kaggle after quick googling: https://www.kaggle.com/c/kkbox-music-recommendation-challenge

**Comment by YY199981474** [comment]
> > > Hi My name is hsuchengmath .
> > > I recently research recommendation system and my Boss told me that
> > > I read MKR the paper . I had read and try to implement it by Pytorch .
> > > But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
> > > After frustration , I search for MKR Pytorch on website and find your GitHub .
> > > Can we be friend ?
> > 
> > 
> > Hi My name is Yiping Li, recently I research recommendation system basing on KGE, do you know how can I get a list which contains such data:
> > Top 10 music data users like
> > do you know ? hsuchengmath.
> 
> Hi neuliyiping, do you mean to get an open dataset of music recommendation? Here is one on Kaggle after quick googling: https://www.kaggle.com/c/kkbox-music-recommendation-challenge


Hello, I am currently running your program and have noticed that the top metric is not displaying any results. The error is caused by the line "self.head_relation_concat = torch.cat([self.head_embeddings, self.relation_embeddings], 1)". Could you please advise on how to resolve this issue? I would greatly appreciate your response.

**Comment by neuliyiping** [comment]
I'm very sorry, this project is my undergraduate graduation project. The data set can no longer be found. I haven't been engaged in deep learning related work for many years. I'm sorry that I can't help you.





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "hsientzucheng/MKR.PyTorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月27日(星期天) 晚上9:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [hsientzucheng/MKR.PyTorch] Hi hsientzucheng  (#1)





    
Hi My name is hsuchengmath .
 I recently research recommendation system and my Boss told me that
 I read MKR the paper . I had read and try to implement it by Pytorch .
 But unfortunately , Though I finish code , the accuracy is not acheived it that MKR model should achieve (approximately 50%) .
 After frustration , I search for MKR Pytorch on website and find your GitHub .
 Can we be friend ?
  
Hi My name is Yiping Li, recently I research recommendation system basing on KGE, do you know how can I get a list which contains such data:
 Top 10 music data users like
 do you know ? hsuchengmath.
  
Hi neuliyiping, do you mean to get an open dataset of music recommendation? Here is one on Kaggle after quick googling: https://www.kaggle.com/c/kkbox-music-recommendation-challenge
  
Hello, I am currently running your program and have noticed that the top metric is not displaying any results. The error is caused by the line "self.head_relation_concat = torch.cat([self.head_embeddings, self.relation_embeddings], 1)". Could you please advise on how to resolve this issue? I would greatly appreciate your response.
